examples/bzlmod/runfiles/BUILD.bazel

```diff
@@ -1,5 +1,6 @@
 load("@rules_python//python:defs.bzl", "py_test")
 
+# gazelle:ignore
```

Comment:
@aignas I am getting errors when I try to have gazelle create the target and resolve the dependencies for the `"@our_other_module//other_module/pkg:lib",`.

```
$ bazel run //:gazelle update
INFO: Analyzed target //:gazelle (39 packages loaded, 240 targets configured).
INFO: Found 1 target...
Target //:gazelle up-to-date:
  bazel-bin/gazelle-runner.bash
  bazel-bin/gazelle
INFO: Elapsed time: 0.381s, Critical Path: 0.02s
INFO: 1 process: 1 internal.
INFO: Build completed successfully, 1 total action
INFO: Running command line: bazel-bin/gazelle update
gazelle: ERROR: failed to validate dependencies for target "//runfiles:runfiles_test": "other_module.pkg.lib" at line 18 from "runfiles/runfiles_test.py" is an invalid dependency: possible solutions:
        1. Add it as a dependency in the requirements.txt file.
        2. Instruct Gazelle to resolve to a known dependency using the gazelle:resolve directive.
        3. Ignore it with a comment '# gazelle:ignore other_module.pkg.lib' in the Python file.

"other_module.pkg" at line 18 from "runfiles/runfiles_test.py" is an invalid dependency: possible solutions:
        1. Add it as a dependency in the requirements.txt file.
        2. Instruct Gazelle to resolve to a known dependency using the gazelle:resolve directive.
        3. Ignore it with a comment '# gazelle:ignore other_module.pkg' in the Python file.

"other_module" at line 18 from "runfiles/runfiles_test.py" is an invalid dependency: possible solutions:
        1. Add it as a dependency in the requirements.txt file.
        2. Instruct Gazelle to resolve to a known dependency using the gazelle:resolve directive.
        3. Ignore it with a comment '# gazelle:ignore other_module' in the Python file.
 ```

I tried using the resolve directive, but I could not get that working in the manner I assumed would work.  I'm guessing that sub-modules, at this point, are not supported.  Is that correct?

Reply:
@f0rmiga any ideas around the question above ^

Reply:
@aignas this example does not seem to download python when it is running.  It is using the hosts python binary.  If I am correct we have bazel download python by setting the `interpreter`  in pip_parse.

From the WORKSPACE file in the pre bzlmod example

```
pip_parse(
    name = "pip",
    # Generate user friendly alias labels for each dependency that we have.
    incompatible_generate_aliases = True,
    # (Optional) You can provide a python_interpreter (path) or a python_interpreter_target (a Bazel target, that
    # acts as an executable). The latter can be anything that could be used as Python interpreter. E.g.:
    # 1. Python interpreter that you compile in the build file.
    # 2. Pre-compiled python interpreter included with http_archive.
    # 3. Wrapper script, like in the autodetecting python toolchain.
    #
    # Here, we use the interpreter constant that resolves to the host interpreter from the default Python toolchain.
    python_interpreter_target = interpreter,
    # Set the location of the lock file.
    requirements_lock = "//:requirements_lock.txt",
    requirements_windows = "//:requirements_windows.txt",
)
```

What is the syntax that I use in a MODULES.bzl file to load the interpreter?  I noticed this by running this example on windows and it is complaining that it cannot find python in the path, even though python exists on the host.

Reply:
I figured this out. The current parameters will only accept hard coded values and won’t work cross platform